core/app/models/concerns/spree/active_storage_adapter.rb

```diff
@@ -107,6 +107,8 @@ def filename
 
     def url(style = default_style)
       attachment.url(style)
+    rescue ActiveStorage::FileNotFoundError
+      '/images/corrupted-image.png'
```

Comment:
This image is not provided by the system, and I think it shouldn't because the frontend and the backend have separate assets folders while this concern code would serve both of them. One alternative could be using a base64 string here with a standard broken image icon, but it's not ideal as well.

On a side note: what about removing this fix from this PR and opening another one just for the other issue?

Reply:
Removed temporary fix to be brought up in another branch